int32 TIMEOUT   =0
int32 ABORTING  =1
int32 SUCCEEDED =2
int32 ACTIVED =3
int32 DOING =4
int32 PREEMPTED =5
int32 status
int32 targetindex
float32 x
float32 y
float32 theta

